mudlet-mapper.xml

```diff
@@ -5390,7 +5393,7 @@ speedWalkCounter = 0
 mmp.speedWalk = mmp.speedWalk or {}
 mmp.speedWalkPath = mmp.speedWalkPath or {}
 mmp.speedWalkDir = mmp.speedWalkDir or {}
-local newversion = "developer"
+local newversion = "20.11.2"
```

Comment:
Should still be developer, right?

Reply:
Ahh, this is my first PR here. Didn't know that's how that worked. Fixing!